django_freeradius/static/django-freeradius/js/radcheck.js

```diff
@@ -1,8 +1,11 @@
+var django = null;
```

Comment:
no you cannot do this or you will break the application.

Add this to the `predef` globals instead, remove this line.